src/stories/Library/date-range/DateRange.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable import/no-extraneous-dependencies */
+// Import default styling
+import "flatpickr/dist/flatpickr.css";
+
+import React, { useEffect } from "react";
+import { Helmet } from "react-helmet";
+import { BaseOptions } from "flatpickr/dist/types/options";
+import clsx from "clsx";
+
+declare global {
+  interface Window {
+    // Allow attachment of global Flatpickr configuration options.
+    flatpickrOptions: Partial<BaseOptions>;
+  }
+}
+
+type Modifiers = "filter";
+
+export type DateRangeProps = {
+  open?: boolean;
+  modifiers?: Modifiers[];
+};
+
+const DateRange: React.FC<DateRangeProps> = ({ open, modifiers }) => {
+  // Use a set of static values for testing.
+  const now = "2024-01-19";
+  const from = "2024-01-01";
+  const to = "2024-01-10";
+
+  useEffect(() => {
+    window.flatpickrOptions = {
+      now,
```

Comment:
I think it would be nice to use the functionality built in of displaying 2 months instead of one on desktop / when there is room for it. 

Up to you

```suggestion
    window.flatpickrOptions = {
      now,
      showMonths={2}
```